index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile first Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./image/static/Inter-Black.ttf"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
```

Comment:
- [ ] Your portfolio appears to be incomplete. the `toolbar` and  `headline` sections are missing. Kindly add them to your portfolio so that it is more detailed and professional.

![image](https://user-images.githubusercontent.com/98453979/205386262-e4a39663-999a-43eb-b0f9-8bef0734f2fa.png)